rootkeys/fs_test.go

```diff
@@ -0,0 +1,73 @@
+package rootkeys
+
+import (
+	"testing"
+)
+
+func TestRootsFS(t *testing.T) {
```

Comment:
I'm curious since RootsFs is only used by `third_party/tailscale/distsign`, why not move rootsfs to distsign package?

Reply:
> RootsFs is only used by third_party/tailscale/distsign

The `rootkeys/keys` contains the set of public keys managed by lepton, whereas `third_party/tailscale/distsign` is the code we borrowed from tailscale repo

Reply:
The problem is that the dependency is confusing. Referring to the original implementation at `https://github.com/tailscale/tailscale/blob/v1.76.3/clientupdate/distsign`, we only need to override the roots package. Therefore, I’d like to move the rootkeys package to the distsign package as roots, and revert the roots.go file.